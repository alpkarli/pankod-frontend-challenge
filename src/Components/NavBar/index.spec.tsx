import { create } from "react-test-renderer";
import { Navbar } from "@Components";

describe("Navbar", () => {
    it("should match snapshot", () => {
        const component = create(<Navbar />);
        expect(component.toJSON()).toMatchSnapshot();
    })
});