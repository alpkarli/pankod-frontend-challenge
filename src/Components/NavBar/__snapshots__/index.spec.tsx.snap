// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<div
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <button
        className="button--link button "
        href="/home"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <p
          className="header__title"
        >
          DEMO Streaming
        </p>
      </button>
      <div
        className="header__content-right"
      >
        <button
          className="button--link button"
        >
          Log in
        </button>
        <button
          className="button button--secondary "
        >
          Start your free trial
        </button>
      </div>
    </div>
  </div>
</div>
`;
